### DISCUSSION TITLE
26.3 FromIterator

### DISCUSSION THREAD

**Discussion by njr0** [google/comprehensive-rust#1268] [General]
This is a bit trivial, but the code in 26.2 doesn't print anything. So I suppose running it shows that it doesn't produce an error, but it's a bit underwhelming.

I added
```
println!("{:?}", prime_squares);
```
at the end, which (more convincingly) printed
```
[4, 9, 25, 49]
```

**Comment by djmitche** [comment]
https://github.com/google/comprehensive-rust/pull/1271
